iphone/Classes/MediaModule.m

```diff
@@ -1704,6 +1704,12 @@ - (void)showPicker:(NSDictionary *)args isCamera:(BOOL)isCamera
       [self displayCamera:picker];
     }
   } else {
+    if ([TiUtils isIOS11OrGreater]) {
+      BOOL allowTranscoding = [TiUtils boolValue:@"allowTranscoding" properties:args def:YES];
```

Comment:
I agree with solution 1. Let's have it simple for now as it is. We can add "transcodeFormat" later.
@rlustemberg  If you can resolve other two minor issues-
1. add minimum iOS version in document
2. Use [TiUtils isIOSVersionOrGreater:@"11.0"] instead of [TiUtils isIOS11OrGreater].
we will be good to go. Thanks!

Reply:
Done!